docs/learn.md

```diff
@@ -715,13 +673,10 @@ Two new numeric literal forms are added for binary (`b`) and octal (`o`).
 0o767 === 503 // true
 ```
 
-<blockquote class="alert alert--warning">
-  <h4>Only supports literal form</h4>
-  <p>
-    Babel is only able to transform <code>0o767</code> and not
-    <code>Number("0o767")</code>.
-  </p>
-</blockquote>
+:::caution Only supports literal form
+Babel is only able to transform <code>0o767</code> and not
+<code>Number("0o767")</code>.
```

Comment:
ditto.

Reply:
Im just wondering why these were in new line? May be we should set-up some linting rules for all these?

Reply:
I have no idea. I guess it has to be dealt with how MDX parses the source.

> May be we should set-up some linting rules for all these?

Good idea, I am wondering are there any such rules out there now.

Reply:
https://github.com/mdx-js/eslint-mdx - This is what we are looking for?